### PR TITLE
Feature: key color

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: lib/functions-custom.php [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+  if (is_single() && is_single_type('programacion', $post)) {
+
+    $post_id = $post->ID;
+
+  } else {
+
+    // get post ID from latest programacion. if none fallback
+    $latest_programacion = get_posts(array(
+      'post_type' => 'programacion',
+      'posts_per_page' => -1,
```
Comment (mikeyrayvon) [inline lib/functions-custom.php]: couldnt you just get 1 post instead of all of them? @varffs

File: lib/functions-custom.php [hunk 2]
```diff
@@ -1,3 +1,51 @@
...
+
+  if ($post_id) {
+
+    // post color value from post meta
+    $meta = get_post_meta($post_id, '_igv_color');
+
+    if (!empty($meta[0])) {
+      $key_color = $meta[0];
+    } else {
+      $key_color = 'red';
```
Comment (mikeyrayvon) [inline lib/functions-custom.php]: should we make a note to revise this later?

File: lib/functions-custom.php (hunk 1 above)
Comment (varffs) [inline lib/functions-custom.php]: Oh lol so used to -1 when I wanted 1

File: lib/functions-custom.php [hunk 3]
```diff
@@ -1,3 +1,51 @@
...
+    }
+
+  } else {
+
+    $key_color = 'red';
```
Comment (mikeyrayvon) [inline lib/functions-custom.php]: maybe we should define this variable at the top of the function

**Pull request merged**